src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,385 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This test stress tests bucket unpacking. Each document is large with many top-level fields.
+  Each query targets a measurement, and cannot use the index on time/meta for time-series collections.
+  Each query will target a different number of buckets, and a different number of fields to unpack. 
+  We should see the runtime for each query increase based on the number of buckets and fields to unpack.
+  We also compare the runtime of queries that unpack empty documents (or exclude all fields).
+
+Keywords:
+  - timeseries
+  - aggregate
+
+GlobalDefaults:
+  Database: &database test
+  Collection: &collection Collection0
+  maxPhases: &maxPhases 20
+  Repeat: &repeat 100
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 4000
+      
+ActorTemplates:
+- TemplateName: RunMatchPipeline
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "RunMatchPipeline"}}
+    Type: CrudActor
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "Active", Default: 1000}}]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+          - OperationName: aggregate
+            OperationCommand:
+              Pipeline: [
+                    {
+                      $match:
+                        {
+                          "measurement0":
+                            {
+                              $lte:
```

Comment:
nit: even though it's adding a little more duplication to the actors, I'd use as a parameter the whole predicate, that is `Predicate: {$lte: 6000}`. I think it will help with readability of the individual actors (also, technically speaking, a "predicate" is a boolean operator and not a number).